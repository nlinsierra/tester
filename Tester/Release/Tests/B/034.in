QS AC 6H 9S KD JH AH 9D 7C 8H 2H JD 5C 3H 4D QH JS 9C 8C QC 6S KH 5S 3D 3C AS 4H
