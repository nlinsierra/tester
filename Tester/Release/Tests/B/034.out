25
2C 4C 6C TC JC KC 2D 5D 6D 7D 8D TD QD AD 5H 7H 9H TH 2S 3S 4S 7S 8S TS KS
